app/javascript/components/analytics/widgets/MonitoreeFlow.js

```diff
@@ -22,70 +22,91 @@ class MonitoreeFlow extends React.Component {
           transferred_in: thisTimeFrameData?.transferred_in || 0,
           closed: thisTimeFrameData?.closed || 0,
           transferred_out: thisTimeFrameData?.transferred_out || 0,
+          exposure_to_isolation: thisTimeFrameData?.exposure_to_isolation || 0,
+          isolation_to_exposure: thisTimeFrameData?.isolation_to_exposure || 0,
         };
       });
     });
   }
 
+  renderWorkflowTable(data, index) {
+    let oppositeWorkflow = _.upperCase(WORKFLOWS[Number(index == 0 ? 1 : 0)]);
+    return (
+      <Col lg="12">
+        <table className="analytics-table">
+          <thead>
+            <tr>
+              <th className="py-0"></th>
+              {MONITOREE_FLOW_HEADERS.map((monitoreeFlowHeader, index) => (
+                <th key={index}>{monitoreeFlowHeader}</th>
+              ))}
+            </tr>
+          </thead>
+          <tbody key={index}>
+            <tr>
```

Comment:
can we also add `style={{ height: '25px' }}` here? there's a lot of whitespace at the top of both tables

Reply:
Good call https://github.com/SaraAlert/SaraAlert/pull/834/commits/ce3fdacb48fc1507331a644e46a32b5a69c024b6

Reply:
Good call https://github.com/SaraAlert/SaraAlert/pull/834/commits/ce3fdacb48fc1507331a644e46a32b5a69c024b6